docs/content/docs/ide/Benchmarking.md

```diff
@@ -5,20 +5,21 @@ type: docs
 
 # Бенчмарки IDE
 
```

Comment:
Let's add "avito" disclaimer. 
We have a dedicated shortcode. https://avito-tech.github.io/avito-android/docs/contributing/docs/#internal-links

Reply:
I've added more info about plugin's disfunction